distributed/tests/test_asyncio.py

```diff
@@ -0,0 +1,5 @@
+import sys
+
+
+if sys.version_info > (3, 5):
```

Comment:
So this will be skipped on 3.5 but not on 3.5.1? Perhaps a `>=` operator is what you're looking for here :-)